File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let summ = 0
    let ans = Array(nums.length)
    for (let i = 0; i < nums.length; i++) {
        summ += nums[i]
        ans[i] = summ
    }
    return ans
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int summ = 0;
        int ans[] = new int[nums.length];
        for (int i = 0; i < nums.length; i++) {
            summ += nums[i];
            ans[i] = summ;
        }
        return ans;
    }
}